pkg/provision.go

```diff
@@ -12,6 +12,7 @@ import (
 // Provision interface
 type Provision interface {
 	DecommissionCached(id string, reason string) error
+	GetWorkloadStatus(id string) (gridtypes.ResultState, bool, error)
```

Comment:
Can we add a doc string on top of this to clarify what are the returned values (what the bool is for someone who is reading the code for the first time)